File: tests/regression/matlab_and_octave/util/fix_normalizer_name_inconsistency.m
function y=fix_normalizer_name_inconsistency(name)
	if strcmp(name, 'IdentityKernelNormalizer')==1
		y='IDENTITY';
	elseif strcmp(name, 'AvgDiagKernelNormalizer')==1
		y='AVGDIAG';
	elseif strcmp(name, 'SqrtDiagKernelNormalizer')==1
		y='SQRTDIAG';
	elseif strcmp(name, 'FirstElementKernelNormalizer')==1
		y='FIRSTELEMENT';
	end

File: tests/regression/r_static/util/fix_normalizer_name_inconsistency.R
fix_normalizer_name_inconsistency <- function(name) {
	if (regexpr('Identity', name)>0) {
		return('IDENTITY')
	} else if (regexpr('AvgDiag', name)>0) {
		return('AVGDIAG')
	} else if (regexpr('SqrtDiag', name)>0) {
		return('SQRTDIAG')
	} else if (regexpr('FirstElement', name)>0) {
		return('FIRSTELEMENT')
	} else {
		return(FALSE)
	}
}
